gcp/index.ts

```diff
@@ -0,0 +1,92 @@
+import * as k8s from "@pulumi/kubernetes";
+import * as gcp from "@pulumi/gcp";
+
+import * as config from "./config";
+import * as model from "./model";
+import * as gke from "./gke";
+import * as utils from "./utils";
+
+let cloud: model.Cloud;
+
+if (config.cloud == config.GcpCloud)
+  cloud = new gke.GcpCloud();
+else
+  throw new Error(`invalid cloud configuration: ${config.cloud}`);
+
+let cluster: model.KubernetesCluster = cloud.createKubernetesCluster();
+
+// K8s namespace for operator
+let namespaceName = config.operatorNamespace;
+
+// Output the cluster's kubeconfig and name
+export const kubeconfig = cluster.kubeconfig;
+export const clusterName = cluster.name;
+
+// Create a k8s namespace for operator
+let namespace = new k8s.core.v1.Namespace(namespaceName, {
+  metadata: {
+    // fixme: add to configuration, if DNE let pulumi generate random suffix?
+    // otherwise pulumi will append a random suffix to the namespace.. might be useful for integration testing to do that
+    name: namespaceName 
+  }
+}, {provider: cluster.k8sProvider});
+
+// Operator namespace name
+export const operatorNamespace = namespace.metadata.name;
+
+let serviceAccountName = utils.name("sa");
+let serviceAccount = cloud.operatorServiceAccount(
+  cluster, 
+  serviceAccountName, 
+  namespace);
+
+  // Install the license key into the namespace
+const licenseSecret = new k8s.yaml.ConfigGroup("license-secret",
+  { files: "/Users/debasishghosh/lightbend/akka-cloud-platform-gcp-deploy/license.yaml" },
+  { provider: cluster.k8sProvider },
+);
+
+// Install the GCP Marketplace applications CRD in the cluster
+const appCrd = new k8s.yaml.ConfigGroup("app-crd",
+  { files: "https://raw.githubusercontent.com/GoogleCloudPlatform/marketplace-k8s-app-tools/master/crd/app-crd.yaml" },
+  { provider: cluster.k8sProvider },
+);
+
+// Install Akka Cloud Platform Helm Chart
+// From the Platform Guide:
+//
+// helm install akka-operator akka-operator-helm/akka-operator \
```

Comment:
Rather remove this doc, since it will quickly get outdated, rather link to platforn guide

Reply:
👍